frontend/src/utils/structuredData.ts

```diff
@@ -0,0 +1,74 @@
+import { ProfilePageStructuredData } from 'types/profilePageStructuredData'
+import type { UserDetails } from 'types/user'
+
+/**
+ * JSON-LD structure data for ProfilePage
+ * https://developers.google.com/search/docs/appearance/structured-data/profile-page
+ *
+ * - @context: "https://schema.org"
+ * - @type: "ProfilePage"
+ * - mainEntity: A Person or Organization (using type Person for OWASP community members)
+ *
+ */
+export function generateProfilePageStructuredData(
+  user: UserDetails,
+  baseUrl = 'https://nest.owasp.org'
+): ProfilePageStructuredData {
+  const profileUrl = `${baseUrl}/members/${user.login}`
+
+  const structuredData: ProfilePageStructuredData = {
+    '@context': 'https://schema.org',
+    '@type': 'ProfilePage',
+    mainEntity: {
+      '@type': 'Person',
+      name: user.name || user.login,
+      description: user.bio,
+      image: user.avatarUrl,
+      url: profileUrl,
+      sameAs: [user.url], // GitHub profile URL
+      memberOf: {
+        '@type': 'Organization',
+        name: 'OWASP',
+        url: 'https://owasp.org',
+      },
+    },
+  }
+
+  if (user.company) {
```

Comment:
Use [spread operator](https://github.com/OWASP/Nest/blob/eabb106f08b0b17413355253ac42d698109b3e9e/frontend/src/app/organizations/%5BorganizationKey%5D/layout.tsx#L51) approach for these cases.